src/pages/workspaces/List.js

```diff
@@ -252,7 +258,7 @@ export const WorkspaceList = () => {
           },
           size: { basis: 120, grow: 1, shrink: 0 }
         }, {
-          headerRenderer: () => null,
+          headerRenderer: () => div({ className: 'sr-only' }, ['Actions']),
```

Comment:
This addresses a specific issue identified by aXe in which a column header was missing text. In this case the text will only appear for screen reader users